### DISCUSSION TITLE
More Options in Account expand

### DISCUSSION THREAD

**Discussion by Chiller2019** [powerpaul17/nc_money#20] [Ideas]
Hey,

I wish a Option when I expand a payment.
![image](https://user-images.githubusercontent.com/50303399/198846748-2fd4c252-4006-4216-bd89-c073329144a4.png)

So a ex.
On account under income, I create a bill for Underground.
+75 for the account, there is a option to reduce another account but what I wish is to + money on a another account.
Ex. for the account "Finanzen", that the financial assets for a semester in my ex.

Calculation:
Typ: Income
Account: Veranstaltung/event
Amount: Underground
+75€

Typ: Ainancial assets
Account: Finanzen/finances
Amount: Underground
+75€

Kind regards

**Comment by powerpaul17** [comment]
Hello, if I understand correctly, what you want is having +75€ in "Finanzen" and -75€ in "Veranstaltungen". The transaction itself has to be balanced, i.e. the values of the splits sum up to 0. The not intuitive problem here is that "Erträge" (and "Verbindlichkeiten") will have a negative value.

**Reply by Chiller2019** [reply]
> Hello, if I understand correctly, what you want is having +75€ in "Finanzen" and -75€ in "Veranstaltungen". The transaction itself has to be balanced, i.e. the values of the splits sum up to 0. The not intuitive problem here is that "Erträge" (and "Verbindlichkeiten") will have a negative value.

Maybe my calculation was a mistake to understand.
So:
Under "Erträge/Veranstaltungen" is a positive balance with +75€ from Event "Underground". (We eraned on this event money)
So, in "Erträge/Veranstaltungen" shall still be a positive amount.
And now, I wish a Option to add this balance also, with +75 on "Vermögenswerte/Finanzen", because "Vermögenswerte/Finanzen" is our 6-month-period-government grant + our money grant from events or simillar.


Bit clearer? I set a nwe pic for better vision.
![image](https://user-images.githubusercontent.com/50303399/198873484-4ce35119-0c29-473f-a00d-8b24b20959f5.png)

**Reply by powerpaul17** [reply]
Yes, I understood. Can you have a look at #18? I explained it there too..

This shows more of what I mean: [https://www.quora.com/Why-do-income-accounts-have-negative-signs-in-ledgers?share=1](https://www.quora.com/Why-do-income-accounts-have-negative-signs-in-ledgers?share=1)
